mappings/net/minecraft/world/World.mapping

```diff
@@ -1,14 +1,25 @@
 CLASS none/aiu net/minecraft/world/World
+	FIELD B lootTableHandler Lnone/bbj;
 	FIELD C profiler Lnone/os;
-	FIELD E isRemote Z
+	FIELD E isClient Z
```

Comment:
isRemote is more factually correct, IMHO. You could theoretically have a remote world which is not just an instance of WorldClient.